github/client.go

```diff
@@ -685,12 +694,53 @@ func (client *Client) newOctokitClient(auth octokit.AuthMethod) *octokit.Client
 	host = normalizeHost(host)
 	apiHostURL := client.absolute(host)
 
-	httpClient := newHttpClient(os.Getenv("HUB_TEST_HOST"), os.Getenv("HUB_VERBOSE") != "")
+	tlsConfig, err := client.newTLSConfig()
+	if err != nil {
+		panic(err)
```

Comment:
I put this `panic` here as a placeholder, but it may be desirable to refactor this method to return an error.

Reply:
There's also `utils.check(err)` which will quit hub if `err` is not nil.

Reply:
Ah yes, great point! I changed over to use that in f8cb480. Thanks.